### DISCUSSION TITLE
Android and iOS Client fork

### DISCUSSION THREAD

**Discussion by ghost** [GFW-knocker/gfw_resist_tls_proxy#88] [Ideas]
https://github.com/yebekhe/DPI-TLS-Tunnel

Android and iOS GUI , Based on LoadBalancer brach

**Comment by GFW-knocker** [comment]
good job
thank you so much for your effort
please add instruction for ios users
then i mention you in main page

**Reply by ghost** [reply]
By a mistake I thought that creating iOS package is like android but after exporting android apk I realized that making ios package from my code needs macOS and currently I don't have MacBook to try exporting iOS version
